.ddev/docker-compose.environment.yaml

```diff
@@ -4,6 +4,6 @@ services:
     environment:
       - APP_ENV=dev
       - APP_SECRET=ddf011a53a72d4f85e6fcfd4d1a72737
-      - DOCS_ROOT_PATH=../docs_server/docs.typo3.org/Web
+      - DOCS_ROOT_PATH=/var/www/html/docs_server/docs.typo3.org/Web
```

Comment:
This is DDEV-related (local environment) configuration and it won't impact anything else. If i remember correctly the idea was to make path absolute for local environment because if you ran a command in different dir than intended it broke the command.

I'm not attached to this solution, just wanted to improve local environment user experience.